tests/testthat/test-a-recording.R

```diff
@@ -43,3 +43,41 @@ test_that("We can specify the path when starting capture", {
   dbDisconnect(con)
   stop_db_capturing()
 })
+
+test_that("dbGetQuery error checking", {
+  # Check to make sure con was not created before start_db_capturing()
+  suppressMessages(con <- nycflights13_create_sqlite(verbose = FALSE))
+
+  start_db_capturing()
+
+  # Testthat sets this to "tests/testthat//_memory_"
+  # Setting this to NULL so it will mimic a developers experience
+  .dittodb_env$db_path <- NULL
+
+  regex_db <- "^Database capture failed"
+
+  error_get_query <- expect_error(
+    object = dbGetQuery(con, "SELECT * FROM airlines"),
+    regexp = regex_db
+  )
+
+  testthat_transition(
+    old = expect_error(dbListTables(con)),
+    new = expect_error(expect_error(dbListTables(con)), regex_db)
```

Comment:
Would you mind asserting that these show `"Database capture failed"`?

It might not be possible with the old version given the way that error catching used to work. If you can't get it working that's totally fine, we can keep it as is.

Reply:
Ah whoops, I totally didn't even try to add it to the other ones for some reason haha. I added the `regex_db` to all asserts/expect_errors :) all checks have passed as well